.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: VS Code extension CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      max-parallel: 3
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-18.04]
+        node-version: [8.x]
+        #node-version: [8.x, 10.x]
```

Comment:
why not using the latest node version?

Reply:
good point, I'll try with the latest - shouldn't make much difference just for the CI

Reply:
updated with node 13.x - build still passing